File: src/main/java/br/com/cvc/scheduling/transfer/repository/TransferRepository.java
package br.com.cvc.scheduling.transfer.repository;

import br.com.cvc.scheduling.transfer.model.Transfer;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

/**
 * Transfer Repository
 *
 * @author Victor Rodrigues de Matos
 */
@Repository
public interface TransferRepository extends JpaRepository<Transfer,Long> {

    List<Transfer> findAllBySourceAccount(String sourceAccount);
}

File: src/main/java/br/com/cvc/scheduling/transfer/repository/TransferRepository.kt
package br.com.cvc.scheduling.transfer.repository

import br.com.cvc.scheduling.transfer.model.Transfer
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * Transfer Repository
 *
 * @author Victor Rodrigues de Matos
 */
@Repository
interface TransferRepository : JpaRepository<Transfer, Long> {

    fun findAllBySourceAccount(sourceAccount : String) : List<Transfer>;
}